leadbutt.py

```diff
@@ -160,9 +327,20 @@ def get_metric_statistics(**kwargs):
         connection = kwargs.pop('connection')
         return connection.get_metric_statistics(**kwargs)
 
+    def get_logs_statistics(**kwargs):
+        """
+        A thin wrapper around boto.logs.get_log_events, for the
+        purpose of adding the @retry decorator
```

Comment:
The retry issues with regular CloudWatch were pretty hard to diagnose, better to fix the comment (and move the function to a more usual location) if it is not needed, or add the decorator with the parsed args, as done with regular CloudWatch metrics,